core/ballerina/.devcontainer.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Let's remove this file. update gitignore

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-health.dicom/pull/2/commits/8315c0ae95c0d34c5011021e79467e1409465c54